web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -638,6 +642,12 @@
   })();
 
   window.liftJQuery = {
+    onEvent: function(elementOrId, eventName, fn) {
+      if (typeof elementOrId == 'string')
+        elementOrId = '#' + elementOrId;
+
```

Comment:
Yeah, completely understand—I use them 99% of the time as well. I pulled in some code for event handling and did some rearrangement here, so it must have just slipped through :)